atlasdb-timelock-server/build.gradle

```diff
@@ -0,0 +1,62 @@
+buildscript {
+    repositories { jcenter() }
+    dependencies {
+        classpath 'com.palantir.gradle.javadist:gradle-java-distribution:0.5.2'
+    }
+}
+
+apply from: "../gradle/shared.gradle"
+
+apply plugin: 'com.palantir.java-distribution'
+apply plugin: 'org.inferred.processors'
+
+
+dependencies {
+    compile project(':lock-impl')
+    compile project(':leader-election-impl')
+    compile project(':atlasdb-config')
+    compile project(':atlasdb-hikari')
+
+    compile "io.dropwizard:dropwizard-core:" + libVersions.dropwizard
+    compile 'com.google.dagger:dagger:2.0.2'
+
+    compile "org.rocksdb:rocksdbjni:4.1.0"
```

Comment:
Actually, I do now have an idea why it is. This is the server, it's going to need to know about the different KVSes to be useful otherwise it won't understand the config you give it.